generators/utils.js

```
@@ -143,8 +143,10 @@ function rewrite(args) {
     });
 
```

Looking at the code, I would say that the needle was not found:
```
    If (otherwiseLineIndex === -1) {
        console.warn(`Needle ${args.needle} not found at file ${args.file}`);
        return args.haystack;
    }
```

---

is it a suggested code ? instead of my change ?

---

Yes, but if you think it’s a different problem, you can go ahead and merge as it is.

---

Looking at the code, I think you're right. I'll apply your change tomorrow, unless you want to do it :)
Thanks a lot

---

Maybe it’s a good idea to throw instead of just log.
Looks an error to me.